python/lsst/pipe/tasks/postprocess.py

```diff
@@ -562,6 +562,10 @@ def addCalibColumns(self, catalog, exposure, exposureIdInfo, **kwargs):
         measureConfig = SingleFrameMeasurementTask.ConfigClass()
         measureConfig.doReplaceWithNoise = False
 
+        # Clear all slots, because we aren't running the relevant plugins.
+        for slot in measureConfig.slots:
+            setattr(measureConfig.slots, slot, None)
+
```

Comment:
Do we not need any of the slots at all?  I guess this is just the recalibration plugins, so no.  And if it runs in ci_hsc then it's fine, I suppose!

Reply:
These slots are what the plugins use to define what gets assigned where, but if the input catalog already has slots, these aren't necessary. And yes, none of them are needed because none of the relevant plugins are being run.